### ISSUE TITLE
Possible bug in constraint

### ISSUE THREAD

**Issue by meatballs** [PyconUK/ConferenceScheduler#60]
As noted by @alexwlchan in #54

L45 conference_scheduler/lp_problem/constraints.py
```
    if events[event].tags is not []:
```

Did you mean to test identity here, or just equality? Since the second half of the comparison will be created every time, I think this always returns True:

```
>>> [] is not []
True
>>> x = []
>>> x is not []
True
```

**Comment by drvinceknight** [comment]
Yup, my bad, should have been equality. Fixing it now (this doesn't break anything, it just means we haven't shaved off some of the constraints).

**Comment by meatballs** [comment]
Fixed by #61
